jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
@davidhalter  you are obviously right. Now I have no idea to fix my particular issue (renaming opens a new split of the same file) while allowing renaming over multiple files.  I'll try to find a fix but it might take some time.